src/components/ProgressIndicator/ProgressIndicatorItem.js

```diff
@@ -0,0 +1,75 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import Body from 'components/Body';
+import Link from 'components/Link';
+import {getBEMClassName} from 'utils';
+
+import CompletionMark from './CompletionMark';
+
+const ProgressIndicatorItem = ({
+  text,
+  href,
+  isActive,
+  isCompleted,
+  canNavigateTo,
+  isApplicable,
+  fixedText = null,
+}) => {
+  const getLinkModifiers = (isActive, isApplicable) => {
+    return [
+      'inherit',
+      'hover',
+      isActive ? 'active' : undefined,
+      isApplicable ? undefined : 'muted',
+    ].filter(mod => mod !== undefined);
+  };
+
+  return (
+    <div className={getBEMClassName('progress-indicator-item')}>
+      <div className={getBEMClassName('progress-indicator-item__marker')}>
+        <CompletionMark completed={isCompleted} />
+      </div>
+      <div className={getBEMClassName('progress-indicator-item__label')}>
+        {isApplicable && canNavigateTo ? (
+          <Link
+            to={href}
+            placeholder={!canNavigateTo}
```

Comment:
`placeholder` will never be `true` because when `canNavigateTo` is `false`, line 35 prevents this branch from rendering.

So this definitely requires more stories:

1. A story with `isApplicable: true, canNavigateTo: true`
2. A story with `isApplicable: true, canNavigateTo: false`
3. A story with `isApplicable: false, canNavigateTo: false`

I don't think that `isApplicable: false, canNavigateTo: true` makes sense, so that should probably `throw new Error(...)` if that's detected.